question-service/src/app.ts

```diff
@@ -4,6 +4,9 @@ import cors from 'cors';
 import * as dotenv from 'dotenv';
 import questionsRouter from './routes/questionsController';
 import userQuestionsRouter from './routes/userQuestionsController';
+import insertQuestionsRouter from './routes/insertQuestionsController';
```

Comment:
These imports should be removed if not cannot build

Reply:
Yups I was trying to remove them just now